homework-solutions/js/hw-4/task-1.js

```diff
@@ -1,5 +1,11 @@
 /*
 
- - Создайте переменную salary со значением 1000
-  - Создайте переменную grade, которая должна получить значение "middle" если salary больше или равна 1000, и значение "junior" - если меньше
+ - РЎРѕР·РґР°Р№С‚Рµ РїРµСЂРµРјРµРЅРЅСѓСЋ salary СЃРѕ Р·РЅР°С‡РµРЅРёРµРј 1000
+  - РЎРѕР·РґР°Р№С‚Рµ РїРµСЂРµРјРµРЅРЅСѓСЋ grade, РєРѕС‚РѕСЂР°СЏ РґРѕР»Р¶РЅР° РїРѕР»СѓС‡РёС‚СЊ Р·РЅР°С‡РµРЅРёРµ "middle" РµСЃР»Рё salary Р±РѕР»СЊС€Рµ РёР»Рё СЂР°РІРЅР° 1000, Рё Р·РЅР°С‡РµРЅРёРµ "junior" - РµСЃР»Рё РјРµРЅСЊС€Рµ
   */
+
+
+  
+let salary = 1000;
+let grade;
+grade = salary >= 1000 ? (grade = "middle") : (grade = "junior");
```

Comment:
В зависимости от условия в `grade` будет присвоено значение, в твоем варианте вьіполняется избьіточное присваивание 
```suggestion
grade = salary >= 1000 ? "middle" : "junior";
```